### ISSUE TITLE
Full width images need basic styling

### ISSUE THREAD

**Issue by lb-** [wagtail/guide#115]
Currently the full width images are not constrained with any styles so they overflow the container and sometimes the viewport.

It would be good to add some basic styling to images to avoid this.

I noticed we have commented out the Bootstrap images styles so not sure on the best approach. 
* https://getbootstrap.com/docs/5.2/content/images/
* https://github.com/wagtail/guide/blob/eebf49eb474a37a0f0718bbdc1e1025c007c2e1e/apps/frontend/static_src/scss/main.scss#L18

## Potential solution

We could just solve the specific problem of these images only via the below SCSS, but not sure where to add it.

```scss
.richtext-image.full-width { 
  width: 100%;
  height: auto;
}
```

### Without styles applied

<img width="1486" alt="Screen Shot 2022-10-05 at 8 03 39 am" src="https://user-images.githubusercontent.com/1396140/193938589-75d4a1d6-90f3-4a51-b801-0b32e227a72d.png">

<img width="636" alt="Screen Shot 2022-10-05 at 8 04 04 am" src="https://user-images.githubusercontent.com/1396140/193938615-684015a8-8ab4-48a3-89ec-b606b3d260d4.png">

### With styles applied (temp in browser)
<img width="1480" alt="Screen Shot 2022-10-05 at 8 03 18 am" src="https://user-images.githubusercontent.com/1396140/193938548-75410348-eca6-4dc3-9b86-a05587aea49d.png">

<img width="638" alt="Screen Shot 2022-10-05 at 8 04 24 am" src="https://user-images.githubusercontent.com/1396140/193938657-494fe2f1-70ed-49a3-bcd4-47e45bba84d9.png">

**Comment by VictoriaAjala** [comment]
Hi @lb- , would you mind if i work on this?

**Comment by lb-** [comment]
Go for it @VictoriaAjala

**Comment by VictoriaAjala** [comment]
Thanks @lb-

**Comment by VictoriaAjala** [comment]
@lb- I can't seem to reproduce the error on the main branch. The main branch isn't the same as the one deployed.
My local version is different from the Guide site at: https://wagtail-guide-staging.herokuapp.com
![Screenshot 2022-10-18 at 19-27-09 Finding your way around - Guide staging](https://user-images.githubusercontent.com/102407780/196515186-fd238db1-cb9b-4ed0-9182-e5b6cd0e7072.png)
![Screenshot 2022-10-18 at 19-15-15 Wagtail user guide](https://user-images.githubusercontent.com/102407780/196515238-4340a833-b813-4423-9ff8-bad0328f94fa.png)

/en-latest/how-to/finding-your-way-around/

**Comment by lb-** [comment]
@VictoriaAjala this is because the public website has different content. 

This Wagtail site's code is for the database models, the fields, and page structure but not the page content itself. The page content is hosted on the database that runs the public site. 

When you get started locally, the script that sets up some initial content (for local testing) is not the same as what's on the public site. 

However, you should be able to add an image in your local running version, see it on your local page view and then debug from there. 

The SCSS change will be something that gets committed to this code and will end up on the public site.

**Comment by Scotchester** [comment]
This was addressed in #154. Current styling on production:

```css
.richtext-image.full-width {
  display: block;
  width: 100%;
  height: auto;
  margin-top: 40px;
}
```
